deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/ProcessorUtils.kt

```diff
@@ -0,0 +1,15 @@
+package com.airbnb.deeplinkdispatch
+
+object ProcessorUtils {
+    @JvmStatic
+    fun decapitalize(str: String?): String? {
+        return if (str == null || (str.length > 1 && str[1].isUpperCase() && str[0].isUpperCase())) {
```

Comment:
I don't understand why decapitalize does nothing when the first are uppercase. Seems worth a comment, or a different name.

fun `String.decapitalize` feels more semantic for kotlin

Reply:
It does not do anything if the first two are capital. 
Renamed it and added a test.